app/controllers/physical_server_controller.rb

```diff
@@ -33,7 +33,7 @@ def show_list
   end
 
   def textual_group_list
-    []
+    [%i(properties relationships assets)]
```

Comment:
Wait, looks like `textual_group_properties` and `textual_group_relationships` are already merged here https://github.com/ManageIQ/manageiq-ui-classic/pull/1295/commits/90e3a5afc6fcc6aa1205b21f119d1868d2faaccd
Ideally, that commit should have had `[%i(properties relationships)]`

Could you make a separate PR that would just have `[%i(properties relationships)]` in `textual_group_list` as a follow-up to https://github.com/ManageIQ/manageiq-ui-classic/pull/1295.

/cc @GabrielSVinha